tutorials/cuda_mps/README.md

```diff
@@ -185,6 +205,83 @@ We vary the number of parallel inferences to show that MPS may not be beneficial
 
 As the number of parallel inferences increases, so does the workload, and the benefit of MPS is more evident. However, when the workload is low, CUDA MPS may not be beneficial. 
 
-| Maximum Latency for 5 Instances | 
+| Maximum Latency for 5 Instances |
+| :-------------------------:|
+![max latency for 5 parallel inf](images/igx-dgpu/parallel_inf_Maximum%20Latency%20(ms).png)|
+
+###  IGX Orin w/ Integrated GPU
+#### MPS Setup on IGX-iGPU
+> Note that we run all commands as root
+
+**1. Please add cuda-12.5+ to `$PATH` and `$LD_LIBRARY_PATH`**
```

Comment:
In case of multiple versions, could we add a check to verify that the expected CUDA version is found? I.e. `nvcc --version` or similar

Reply:
Addressed this.